test/Plutip/Common.purs

```diff
@@ -56,6 +57,8 @@ config =
   , suppressLogs: true
   , customLogger: Nothing
   , hooks: emptyHooks
+  , clusterConfig:
+      { slotLength: Seconds 0.1 }
```

Comment:
Will making it lower affect the tests speed?

Reply:
`plutip-test` not really 

`staking-test` yes:
`slotLength` 0.1 Seconds : 169.28 seconds
`slotLength` 0.01 Seconds : 82.36 seconds

Reply:
However it does get a little buggy at 0.01 with some of the tests. I think 0.05 is a good compromise.

Reply:
> a little buggy

In particular? Flaky tests?

Reply:
Flaky tests.